app/config/config.server.ts

```diff
@@ -8,14 +10,19 @@ interface Config {
 
 let instance: Config | undefined = undefined;
 
+const oidcClientSecretFilePath =
```

Comment:
I considered passing this hardcoded path as ENV var, but in my experience I've never changed this value, so I just hardcoded it. @manuelpuchta  What do you think?

Reply:
This is great, let's ship it and test it.

ℹ️ If it is working, let us update that part on the platform docs [guidelines](https://platform-docs.prod.ds4g.net/user-docs/how-to-guides/ci-cd/seal-secret#use-sealedsecret-as-environment-variable) "Use SealedSecret as environment variable".